cmd/controller/app/options/options.go

```diff
@@ -311,6 +313,9 @@ func (s *ControllerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.ACMEHTTP01SolverResourceLimitsMemory, "acme-http01-solver-resource-limits-memory", defaultACMEHTTP01SolverResourceLimitsMemory, ""+
 		"Defines the resource limits Memory size when spawning new ACME HTTP01 challenge solver pods.")
 
+	fs.BoolVar(&s.RunAsNonRoot, "run-as-non-root", defaultRunAsNonRoot, ""+
+		"Defines the ability to run the http01 solver as root for troubleshooting issues")
+
```

Comment:
suggestion: From looking at the name of this flag, it's not super clear what I'm requesting by passing `--run-as-non-root` to the Controller. We have `acme-http01-solver-resource-limits-memory` above.

What do you think about using `acme-http01-solver-run-as-non-root`? It's wordy, but it's very clear about what's happening.

Reply:
Yep! will adjust this.  considering this as well - is there a place where you document the flags?

Reply:
The flags are documented [on the website](https://cert-manager.io/docs/cli/controller/) but the flow for that page to be updated is separate to the flow for merging a PR for cert-manager here. That page would be updated when this flag is released, which is likely to be when we release cert-manager 1.11 early next year!